src/driver/driver_api.cc

```diff
@@ -288,6 +288,10 @@ IRModule ScheduleToModule(te::Schedule sch, const Array<ObjectRef>& args, const
   tir::PrimFunc f = te::SchedulePostProcToPrimFunc(out_arg_list, std::move(stmt), out_binds);
   f = WithAttr(std::move(f), "global_symbol", runtime::String(name));
 
+  // Mark this schedule as being converted from an TE schedule. Makes sure that
+  // the correct TE passes are run.
+  f = WithAttr(std::move(f), "from_legacy_te_schedule", Bool(true));
```

Comment:
Shall we also update the `CreatePassList` method? The "phase 1" there still uses a boolean flag to create different code paths, and ideally we should actually unify the two paths

Reply:
Whoops, I forgot the most important part of this change. I've removed the is_te_schedule from CreatePassList.